snail-kotlin/src/main/java/com/compass/snail/IObservable.kt

```diff
@@ -17,4 +17,6 @@ interface IObservable<T> {
     fun throttle(delayMs: Long): Observable<T>
     fun debounce(delayMs: Long): Observable<T>
     fun skip(first: Int): Observable<T>
+    fun <U> map(mappingFunction: (T) -> U): Observable<U>
```

Comment:
lets add this on Variable for consistency https://github.com/UrbanCompass/Snail/blob/master/Snail/Variable.swift#L40